### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by VojtechBartoska**
## Description of Change

Adding generic Code of Conduct created by GitHub to this repository. (It's recommended for any sizes projects on Github)

It's well written and covers all different scenarios.

In other commit, I will update the Readme with short description and link to this file.

## Tests scenarios
N/A

## Related links
N/A

### CODE REVIEW

**Review by pedrominatel — APPROVED** [review]
Thanks for using this CoC!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+**Community Impact**: A serious violation of community standards, including
+sustained inappropriate behavior.
+
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time. No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a permanent ban.
+
+### 4. Permanent Ban
```
Comment (pedrominatel) [inline CODE_OF_CONDUCT.md:106]: How does it work for GitHub?

Comment (VojtechBartoska) [reply CODE_OF_CONDUCT.md:106]: Actually good question, I was thinking that we don't need this part and I believe we will not ban (block) anyone in the future. (except some bots/fake accounts).

If I'm correct, you can block the user by whole organization or also just from repository by Project owner. Just what I read on GitHub docs, never used it. :)

Comment (VojtechBartoska) [reply CODE_OF_CONDUCT.md:106]: TBH, this whole Code of Conduct is quite formal and a bit of overkill for our repository. On the other hand, GitHub is more aware of any scenarios and have more experiences, also it was checked by someone more educated and aware in this area.

By adding this to our repo, we kind of agree with terms proposed by Github, what I think is generally reasonable approach.

**Comment by pedrominatel** [comment]
Any comments @me-no-dev? Otherwise, we can move on and merge this and #7529.

**Pull request merged**